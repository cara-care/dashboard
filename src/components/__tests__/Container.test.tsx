import React from 'react';
import { render } from '@testing-library/react';
import Container from '../Container';

describe('<Container />', () => {
  it('renders correctly', () => {
    const { container } = render(
      <Container>
        <span>Test</span>
      </Container>
    );
    expect(container).toMatchSnapshot();
  });
});
