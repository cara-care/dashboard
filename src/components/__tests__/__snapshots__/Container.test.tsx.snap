// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> renders correctly 1`] = `
<div>
  <div
    class="makeStyles-root-1"
  >
    <span>
      Test
    </span>
  </div>
</div>
`;
